sematic/db/migrate.py

```diff
@@ -318,6 +318,12 @@ def dump_schema(file: str):
         + ";\n"
     )
 
+    # make sure the file is created in the workspace, and not in the bazel runtime dir
```

Comment:
Rather than this, can we make it so the `main` has:

```
build_working_dir = os.getenv("BUILD_WORKSPACE_DIRECTORY")
if build_working_dir is not None:
    os.chdir(build_working_dir)
else:
    import sematic
    os.chdir(pathlib.Path(sematic.__file__).parent.as_posix())
```
? This way the CLI is always running from a consistent known directory regardless of whether it was executed with bazel or not.

Reply:
Good point. Added #462 as this would require some non-negligible manual testing, and this is low priority.